.openshift-ci/gcp/create-vm.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+createGCPVM() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_IMAGE_PROJECT="$1"
+    shift
+    local GCP_IMAGE_FAMILY="$1"
+    shift
+    local GCP_SERVICE_ACCOUNT="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+            echo "error: missing parameter GCP_VM_NAME" && return 1
+    [ -z "$GCP_IMAGE_FAMILY" ] && \
+        echo "error: missing parameter GCP_IMAGE_FAMILY" && return 1
+    [ -z "$GCP_IMAGE_PROJECT" ] && \
+        echo "error: missing parameter GCP_IMAGE_PROJECT" && return 1
+    [ -z "$GCP_SERVICE_ACCOUNT" ] && \
+        echo "error: missing parameter GCP_SERVICE_ACCOUNT" && return 1
+
+    success=false
+    if gcloud compute instances create \
+        --image-family "$GCP_IMAGE_FAMILY" \
+        --image-project "$GCP_IMAGE_PROJECT" \
+        --service-account="$GCP_SERVICE_ACCOUNT" \
+        --scopes="storage-rw,cloud-platform"\
+        --machine-type e2-standard-2 \
+        --labels="stackrox-kernel-crawler-osci=true,stackrox-osci-job=${JOB_ID}" \
+        --boot-disk-size=20GB \
+        "$GCP_VM_NAME"; then
+        success=true
+        #break
+    else
+        gcloud compute instances delete "$GCP_VM_NAME"
+    fi
+
+    if test ! "$success" = "true"; then
+        echo "Could not boot instance."
+        return 1
+    fi
```

Comment:
Can't we do this instead?
```suggestion
    if ! gcloud compute instances create \
        --image-family "$GCP_IMAGE_FAMILY" \
        --image-project "$GCP_IMAGE_PROJECT" \
        --service-account="$GCP_SERVICE_ACCOUNT" \
        --scopes="storage-rw,cloud-platform"\
        --machine-type e2-standard-2 \
        --labels="stackrox-kernel-crawler-osci=true,stackrox-osci-job=${JOB_ID}" \
        --boot-disk-size=20GB \
        "$GCP_VM_NAME"; then
        echo "Could not boot instance."
        gcloud compute instances delete "$GCP_VM_NAME"
        return 1
    fi
```

Reply:
We could, but this wont work with retries.